src/client/components/ProductsComponent/ProductsComponent.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import './ProductsComponent.styles.css';
+import Product from '../Product/Product';
+import SortComponent from '../SortComponent/SortComponent';
+
+export default function ProductsComponent({ products }) {
+  const [sortedProducts, setSortedProducts] = useState(products);
+
+  const addFavorites = () => {
+    console.log('added to favorites');
+  };
+  const addToCart = () => {
+    console.log('added to cart');
+  };
+  return (
+    <div className="products-component">
+      <SortComponent
+        products={products}
+        setSortedProducts={setSortedProducts}
+      />
+
+      <div>
+        <ul className="product-list">
```

Comment:
I would create one abstraction here in the form of a Products component here.

Reply:
Okay ..